src/AuthReg.cc

```diff
@@ -45,10 +42,6 @@ Auth::Init()
 #if HAVE_AUTH_MODULE_NEGOTIATE
     static const char *negotiate_type = Auth::Negotiate::Scheme::GetInstance()->type();
     debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << negotiate_type << "'");
-#endif
-#if HAVE_AUTH_MODULE_NTLM
-    static const char *ntlm_type = Auth::Ntlm::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << ntlm_type << "'");
 #endif
     debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication.");
```

Comment:
Since this debugs() is now lying -- some authentication initialization happens later after PR changes -- and since some of us think it has wrong level, I suggest removing it completely:

```suggestion
```